lib/common_test/src/ct_logs.erl

```diff
@@ -1500,6 +1518,16 @@ make_one_index_entry1(SuiteName, Link, Label, Success, Fail, UserSkip, AutoSkip,
 			end,
 		{UserSkip+AutoSkip,integer_to_list(UserSkip),ASStr}
 	end,
+
+	ElapsedTimeStr =
+	if ElapsedTime == undefined ->
+		"";
+	true ->
+		["<td align=right>",
+		float_to_list(ElapsedTime / 1000000, [{decimals, 3}]),
+		"s</td>\n"]
```

Comment:
fix indentation. we prefer emacs style.
![image](https://github.com/erlang/otp/assets/4620572/a07a9873-ae72-4600-bcb7-d99b8f1d7942)

Reply:
Fixed